lib/server/services/eosReport/EosReportService.js

```diff
@@ -108,7 +112,7 @@ class EosReportService {
             },
             [],
             // Use only existing tags
-            (await getTagsByText(['EoS', reportType])).map(({ text }) => text),
+            (await getTagsByText([...typeSpecificTags])).map(({ text }) => text),
```

Comment:
I think here because you already factorized, you can directly call the function (and I think the `[...]` is redundant)